crates/miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -1223,26 +1223,26 @@ end
 
 #! Increments the delta of the account's nonce.
 #!
-#! Example: If this procedure is called with nonce_delta 3 and again with nonce_delta 2, then the
-#! stored nonce_delta will be 5.
+#! Example: If this procedure is called with nonce_increment 3 and again with nonce_increment 2, then the
+#! stored nonce_increment will be 5.
 #!
-#! Inputs:  [nonce_delta]
+#! Inputs:  [nonce_increment]
 #! Outputs: []
 #!
 #! Where:
-#! - nonce_delta is the value by which the nonce changed.
-export.incr_account_nonce_delta
-    exec.get_account_nonce_delta add mem_store.ACCOUNT_DELTA_NONCE_PTR
+#! - nonce_increment is the value by which the nonce changed.
+export.incr_account_delta_nonce_increment
+    exec.get_account_delta_nonce_increment add mem_store.ACCOUNT_DELTA_NONCE_PTR
 end
 
 #! Returns the delta of the account's nonce.
 #!
 #! Inputs:  []
-#! Outputs: [nonce_delta]
+#! Outputs: [nonce_increment]
 #!
 #! Where:
-#! - nonce_delta is the value by which the nonce changed.
-export.get_account_nonce_delta
+#! - nonce_increment is the value by which the nonce changed.
+export.get_account_delta_nonce_increment
     mem_load.ACCOUNT_DELTA_NONCE_PTR
 end
```

Comment:
I know we've started calling it `nonce_increment`, but I wonder if we should go back to `nonce_delta` (the delta would still be defined as the difference between old an new nonce):

- It would be more consistent with other parts of the account delta.
- It would avoid awkward naming like `incr_account_delta_nonce_increment` and `get_account_delta_nonce_increment`.